install/framework/framework.wxs

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
 	<Product Id="*" 
-             Name="NUnit Framework !(bind.FileVersion.nunit.framework.45.dll)" 
+             Name="NUnit Framework $(var.DisplayVersion)" 
```

Comment:
Hmmm... I never worried about that before. I suppose I assume that installers of alphas and betas are quite on top of things and know what's on their systems. Certainly, there has never been any comment about it in the past.

As an experiment, I installed the ancient NUnit 2.4 beta 2 and the new NUnit.Framework 3.0 beta 1. For 2.4, I could see the version number (2.3.2693) and could figure out that it's beta 2 by looking back at the release notes. For 3.0, I see version number 3.0.5560.36863 so I could figure that out if we listed it somewhere.

Your point is well taken. Maybe we should make this a bit easier.